quickstarts/velero-prometheus/config.yml

```diff
@@ -0,0 +1,65 @@
+
+slug: velero-prometheus
+
+title: Velero (Prometheus)
```

Comment:
should this be `Velero (Prometheus Remote Write)` ?

Reply:
Velero (Prometheus) - per Schmiddy